src/components/elements/Checkbox/style.scss

```diff
@@ -0,0 +1,132 @@
+:local {
+  @import '../../../styles/config';
```

Comment:
Make sense! Curious when scss files will be transfiled. If an app is using compiled CSS files, then using absolute or relative wouldn't matter.. but if components compiled inside of the main app, then absolute paths don't seem to work as you thought.